src/System.Diagnostics.Process/tests/ProcessThreadTests.cs

```diff
@@ -80,16 +79,21 @@ public void TestStartTimeProperty()
                     return;
                 }
 
-                if (ThreadState.Initialized != thread.ThreadState)
+                if (ThreadState.Initialized != thread.ThreadState && ThreadState.Ready != thread.ThreadState)
```

Comment:
No, ready state simply means it's available for the processor to run. It may have already been running.